turbo/jsonrpc/otterscan_trace_operations.go

```diff
@@ -56,18 +58,34 @@ func NewOperationsTracer(ctx context.Context) *OperationsTracer {
 	}
 }
 
-func (t *OperationsTracer) CaptureEnter(typ vm.OpCode, from common.Address, to common.Address, precompile bool, create bool, input []byte, gas uint64, value *uint256.Int, code []byte) {
-	if typ == vm.CALL && value.Uint64() != 0 {
+func (t *OperationsTracer) Tracer() *tracers.Tracer {
+	return &tracers.Tracer{
+		Hooks: &tracing.Hooks{
+			OnEnter: t.OnEnter,
+		},
+		GetResult: t.GetResult,
+		Stop:      t.Stop,
+	}
+}
+
+func (t *OperationsTracer) OnEnter(depth int, typ byte, from common.Address, to common.Address, precompile bool, input []byte, gas uint64, value *uint256.Int, code []byte) {
+	if vm.OpCode(typ) == vm.CALL && value.Uint64() != 0 {
```

Comment:
Related to RPC test failure.

```suggestion
	if depth == 0 {
		return
	}

	if vm.OpCode(typ) == vm.CALL && value.Uint64() != 0 {
```